src/Xamarin.Android.Build.Tasks/Tasks/Desugar.cs

```diff
@@ -38,19 +36,15 @@ public class Desugar : JavaToolTask
 
 		public override bool Execute ()
 		{
-			Log.LogDebugMessage ("Desugar Task");
-			Log.LogDebugMessage ("  JavaPlatformJarPath: ", JavaPlatformJarPath);
-			Log.LogDebugMessage ("  DesugarJarPath: ", DesugarJarPath);
-			Log.LogDebugMessage ("  DesugarExtraArguments: ", DesugarExtraArguments);
-			Log.LogDebugMessage ("  ManifestFile: ", ManifestFile);
-			Log.LogDebugMessage ("  OutputDirectory: ", OutputDirectory);
-			Log.LogDebugMessage ("  InputClassesDirectory: ", InputClassesDirectory);
-			Log.LogDebugTaskItems ("  InputJars: ", InputJars);
-
 			if (!Directory.Exists (OutputDirectory))
 				Directory.CreateDirectory (OutputDirectory);
 
-			return base.Execute ();
+			base.Execute ();
+
+			Log.LogDebugMessage ($"{nameof (Desugar)} Outputs:");
```

Comment:
For the same reason you can (and did!) remove the `Log.LogDebugMessage()`s, you don't need to log the output properties: MSBuild already logs them, e.g.

```
  Task "ResolveSdks" (TaskId:30) (TaskId:77)
    Task Parameter:AndroidSdkPath=/Users/builder/android-toolchain/sdk (TaskId:30) (TaskId:77)
    Task Parameter:AndroidNdkPath=/Users/builder/android-toolchain/ndk (TaskId:30) (TaskId:77)
    Task Parameter:ReferenceAssemblyPaths=/Users/builder/jenkins/workspace/xamarin-android-pr-builder/xamarin-android/bin/Debug/lib/xamarin.android/xbuild-frameworks/MonoAndroid/v1.0/ (TaskId:30) (TaskId:77)
    ResolveSdks Outputs: (TaskId:30) (TaskId:77)
      AndroidSdkPath: /Users/builder/android-toolchain/sdk (TaskId:30) (TaskId:77)
      AndroidNdkPath: /Users/builder/android-toolchain/ndk (TaskId:30) (TaskId:77)
      JavaSdkPath: /usr (TaskId:30) (TaskId:77)
      MonoAndroidBinPath: /Users/builder/jenkins/workspace/xamarin-android-pr-builder/xamarin-android/bin/Debug/lib/xamarin.android/xbuild/Xamarin/Android/Darwin/ (TaskId:30) (TaskId:77)
      MonoAndroidToolsPath: /Users/builder/jenkins/workspace/xamarin-android-pr-builder/xamarin-android/bin/Debug/lib/xamarin.android/xbuild/Xamarin/Android (TaskId:30) (TaskId:77)
    Output Property: JavaSdkDirectory=/usr (TaskId:30) (TaskId:77)
    Output Property: MonoAndroidToolsDirectory=/Users/builder/jenkins/workspace/xamarin-android-pr-builder/xamarin-android/bin/Debug/lib/xamarin.android/xbuild/Xamarin/Android (TaskId:30) (TaskId:77)
    Output Property: MonoAndroidBinDirectory=/Users/builder/jenkins/workspace/xamarin-android-pr-builder/xamarin-android/bin/Debug/lib/xamarin.android/xbuild/Xamarin/Android/Darwin/ (TaskId:30) (TaskId:77)
```

Although the above does point out an issue: MSBuild only prints out `[Output]` properties, and in some places we use `<Output/>` with non-`[Output]` properties, e.g. `ResolveSdks.AndroidSdkPath`, so in `<ResolveSdks/>` it makes sense to have the additional "output" properties.

I don't believe that's the case here: the MSBuild defaults should be fine, and these `LogDebugMessage()` calls shouldn't be needed.